reverie/backend_server/persona/prompt_template/run_gpt_prompt.py

```diff
@@ -1271,7 +1272,8 @@ def get_fail_safe(main_act_dur, truncated_act_dur):
   
   return output, [output, prompt, gpt_param, prompt_input, fail_safe]
 
-
+class Decide_to_Talk(BaseModel):
+  decision: str
```

Comment:
Since this response is just a "yes" or "no", a boolean might be more straightforward in this case